### DISCUSSION TITLE
pre-commit

### DISCUSSION THREAD

**Discussion by lmilbaum** [fedora-infra/anitya#1585] [Q&A]
`pre-commit` is a really cool and valuable tool which is widely used. Would you be willing to consider adopting it?
Take a look at PR #1583. It produced lots of linting issues.

**Comment by Zlopez** [comment]
That sounds like a good addition to the project. Could you create another PR to fix the pre-commit issues? It seems that most of them are something that isn't controlled by either flake8 or black
